### PR TITLE
Added Leaders Forum link, deleted mastodon link, added register, suggested discusssion and poster section

### PR DESCRIPTION
**Pull request by chrismpr86**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -42,7 +42,31 @@ There are a few ways to participate:
 2. Suggest, [vote and comment on discussion topics](https://github.com/RSEAA/RSEAA.github.io/discussions/76), and
 3. Submit a [poster, vote and comment on posters](https://github.com/RSEAA/RSEAA.github.io/discussions/75)
 
-Click here to see [last year's discussion topics](https://github.com/RSEAA/RSEAA.github.io/discussions/43) and [last year's posters](https://github.com/RSEAA/RSEAA.github.io/discussions/32).
+## Register for the Unconference
+
+You can now register for the Unconference!
```
Comment (rowlandm) [inline index.md:47]: Can we change this to: 

Coming soon: you will be able to register for RSEAA23!

Comment (rowlandm) [inline index.md:47]: Can we change this to: 

Coming soon: you will be able to register for RSEAA23!

Comment (rowlandm) [inline index.md:47]: Can we change this to: 

Coming soon: you will be able to register for RSEAA23!

Comment (rowlandm) [inline index.md:47]: Can we change this to: 

Coming soon: you will be able to register for RSEAA23!

**Review by rowlandm — CHANGES_REQUESTED** [review]
Just the one minor change please.

Replace "You can now register for the unconference" as it is too confusing.

File: index.md (hunk 1 above)
Comment (chrismpr86) [reply index.md]: updated, can you see?

**Review by rowlandm — APPROVED** [review]
Thanks!

**Pull request merged**